__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("Accessing invalid path", () => {
+  test("status 404 when path does not exist", () => {
+    return request(app)
+      .get("/topics")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Input not found");
+      });
+  });
+});
+
+describe("/api/topics", () => {
+  test("status 200 return array of topic objects", () => {
```

Comment:
This test block is only checking for the length. You should also check that each result has the necessary properties.